### ISSUE TITLE
Link between material sample and preparation

### ISSUE THREAD

**Issue by cgendreau** [DINA-Web/dina-model-concepts#29]
From @jmacklin : A material sample may be sub-sampled from 1..N times. These sub-samples will likely then become catalogued objects so that is fine, however, a sub-sample of a material sample may be sent on loan for ID and thus again will need an identifier. The problem is that Loans are tied to preparations and a material sample is not a preparation… unless it is ;-)

**Comment by falkogloeckler** [comment]
This should be sufficiently covered in the description of [preparation processes](https://github.com/DINA-Web/dina-model-concepts/blob/falkogloeckler-delayed-update/component_descriptions/preparation_process.md) now.
@cgendreau: If so, this issue could be closed.

**Comment by jmacklin** [comment]
Agreed! So, we now loan physical entities that are typically catalogued but are not required to be.
